nilearn/_utils/niimg_conversions.py

```diff
@@ -33,6 +33,23 @@ def _check_same_fov(img1, img2):
             and np.allclose(img1.get_affine(), img2.get_affine()))
 
 
+def _assert_same_fov(**kwargs):
+    """ Assert the equivalence of all provided images. Parameter names are
+        used to generate user friendly error message.
+    """
+    errors = []
+    for (a_name, a_img), (b_name, b_img) in itertools.combinations(
```

Comment:
It's possible but I wanted to have the best error message possible. Note that this function is called with one or two params.